watertap/unit_models/zero_order/pump_variable_zo.py

```diff
@@ -0,0 +1,133 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of a low pressure pump unit
+"""
+
+from pyomo.environ import Constraint, units as pyunits, Var
+from idaes.core import declare_process_block_class
+
+from watertap.core import build_pt, ZeroOrderBaseData
+from idaes.core.util.constants import Constants
+
+# Some more information about this module
+__author__ = "Akshay Rao"
+
+
+@declare_process_block_class("PumpVariableZO")
+class PumpVariableZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for a low pressure pump with variable efficiency.
+    Low pressure correlation: Kuritza, J. C.(2017). https://doi.org/10.1590/2318-0331.0217170018
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "pump_variable"
+
+        build_pt(self)
+
+        self.lift_height = Var(units=pyunits.m, doc="Lift height for pump")
```

Comment:
```suggestion
        self.lift_height = Var(self.flowsheet().config.time, units=pyunits.m, doc="Lift height for pump")
```

Reply:
Although the current formulation for efficiency seems to be independent of pump head, I think we should index this by time since we'll want this to change as well.

Reply:
I think this was just because wherever they were used in their respective constraints, they were not being called with their indices. However, this should be resolved now.